platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulersConfig.java

```diff
@@ -134,4 +136,6 @@ public record PlatformSchedulersConfig(
         @ConfigProperty(defaultValue = "500") int issDetectorUnhandledCapacity,
         @ConfigProperty(defaultValue = "SEQUENTIAL_THREAD") TaskSchedulerType hashLoggerSchedulerType,
         @ConfigProperty(defaultValue = "100") int hashLoggerUnhandledTaskCapacity,
-        @ConfigProperty(defaultValue = "1000") int completeStateNotifierUnhandledCapacity) {}
+        @ConfigProperty(defaultValue = "1000") int completeStateNotifierUnhandledCapacity,
+        @ConfigProperty(defaultValue = "SEQUENTIAL_THREAD") TaskSchedulerType stateHasherSchedulerType,
+        @ConfigProperty(defaultValue = "1") int stateHasherUnhandledCapacity) {}
```

Comment:
maybe this should be a 2, since a 1 in the old code is effectively a 2 in the new code

Reply:
I mistakenly thought that the different unhandled task capacity logic applied only to `SEQUENTIAL` schedulers. 

Changing this to `2`, to match old behavior.